src/config.rs

```diff
@@ -380,6 +384,9 @@ impl Default for Pool {
             automatic_sharding_key: None,
             connect_timeout: None,
             idle_timeout: None,
+            sharding_key_regex: None,
```

Comment:
It would be nice to add these to pgcat.toml which we use for illustration and documentation purposes.